modules/rosa-installing.adoc

```diff
@@ -76,13 +76,13 @@ Flags:
 Use "rosa [command] --help" for more information about a command.
 ----
 +
-.. Optional: Generate the command completion scripts for the `rosa` CLI. The following example generates the Bash completion scripts for a Linux machine:
+.. Optional: Generate the command completion scripts for the ROSA CLI. The following example generates the Bash completion scripts for a Linux machine:
 +
 [source,terminal]
 ----
 $ rosa completion bash | sudo tee /etc/bash_completion.d/rosa
 ----
-.. Optional: Enable `rosa` command completion from your existing terminal. The following example enables Bash completion for `rosa` in an existing terminal on a Linux machine:
+.. Optional: Enable the `rosa` command completion from your existing terminal. The following example enables Bash completion for `rosa` in an existing terminal on a Linux machine:
```

Comment:
Probably best to phrase this as "Enable command completion for the ROSA CLI from your existing terminal" - this is enabling "command completion" or "Bash completion" rather than enabling completion for the rosa command